tracer/src/Datadog.Trace/Logging/Internal/DatadogLogging.cs

```diff
@@ -207,5 +217,25 @@ internal static string GetLogDirectory()
 
             return logDirectory;
         }
+
+        internal static void CleanLogFiles(string logsDirectory)
+        {
+            var date = DateTime.Now.AddDays(-31);
```

Comment:
Should we make it configurable? Should we also give a max number of files as a config know? (in case for any reasons we generate a lot of files?

Also, and unrelated to the cleanup per se, the agent seem more drastic that we are in terms of rollover I believe? https://docs.datadoghq.com/agent/guide/agent-log-files/?tab=agentv6v7. Should we do the same for our rollover strategy (I haven't looked at what we do)

Reply:
Added the configurable option and indeed updated the default value to 32 so the expected files get deleted on startup if they are 31 days old at least.